manual/python/index.md

```diff
@@ -7,13 +7,14 @@ toc: true
 toc_sticky: true
 ---
 
-ROOT can be used from Python thanks to its Python-C++ bindings, called PyROOT. PyROOT is HEP's entrance to all C++ from Python, for example, for frameworks and their steering code. The PyROOT bindings are *automatic* and *dynamic*: no pre-generation of Python wrappers is necessary.
+With PyROOT you can use ROOT, because of its Python-C++ bindings, from Python. PyROOT is HEP's entrance to all C++ from Python, for example, for frameworks and their steering code. The PyROOT bindings are *automatic* and *dynamic*: no pre-generation of Python wrappers is necessary.
 
-With PyROOT, all the ROOT functionality can be accessed from Python while, at the same time, benefiting from the performance of the ROOT C++ libraries.
+With PyROOT you can acces the full ROOT functionality from Python while benefiting from the performance of the ROOT C++ libraries.
 
 PyROOT is compatible with both Python2 (>= 2.7) and Python3.
 
-> Using PyROOT requires working knowledge of Python. Detailed information about the Python language can be found in the [Python Language Reference](https://docs.python.org/3/reference/){:target="_blank"}.
+> The usage of PyROOT requires in-depth knowledge of Python.<br/>
```

Comment:
I wouldn't say `in-depth knowledge` is required, at least for the most common operations. I think `working knowledge` is fine or alternatively `fair knowledge`.